package.json

```diff
@@ -285,10 +285,12 @@
     "raven-js": "3.27.2",
     "razzle": "3.0.0",
     "razzle-plugin-bundle-analyzer": "1.1.3",
+    "rc-time-picker": "4.0.0-alpha.2",
     "react": "16.12.0",
     "react-anchor-link-smooth-scroll": "1.0.12",
     "react-animate-height": "2.0.17",
     "react-cookie": "1.0.5",
+    "react-dates": "21.5.1",
```

Comment:
Note: we are using that date picker in our projects as well and it works like a charm. It is just a large dependency. Though, I guess we have to live with the fact if we want to be able to handle dates properly.

Reply:
This makes the Travis check for bundlesize fail, together with the rc-time-picker dependency.